orangecontrib/spectroscopy/tests/test_preprocess.py

```diff
@@ -588,14 +588,14 @@ def test_iris(self):
                                         [4.75015528, 3.15366444, 1.46254138, 0.23693223]])
 
 
-class TestCurveShift(unittest.TestCase):
+class TestLinearTransform(unittest.TestCase):
 
     def test_simple(self):
         data = Table.from_numpy(None, [[1.0, 2.0, 3.0, 4.0]])
-        f = CurveShift(amount=1.1)
+        f = LinearTransform(amount=1.1, scale=2.)
```

Comment:
Here we'd need better names for "amount". When there was just a shift, that was obvious.